src/fileformats/caoslexer.cpp

```diff
@@ -201,9 +201,10 @@ void lexcaos(std::vector<caostoken>& v, const char* p) {
 	}
 
 bytestr:
-	if (p[0] == '\0' || p[0] == '\r' || p[0] == '\n') {
-		p++;
+	if (p[0] == '\0') {
 		push_value(caostoken::TOK_ERROR);
+	} else if (p[0] == '\r' || p[0] == '\n') {
+		p++;
```

Comment:
This still needs to create an error token, you can’t have newlines within a bytestr

Reply:
Ah, sorry about that. I added more unit tests to lock in the current behavior in #222. I'll rebase on top of that once those tests are in.